### PR TITLE
GIBCT Calculator component [WIP]

### PR DESCRIPTION
**Pull request by ben-USDS-damman**
No reviews yet please; just for visibility into the work.

### CODE REVIEW

File: src/js/gi/components/AccordionItem.jsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ class AccordionItem extends React.Component {
     const expanded = this.state.expanded;
     return (
       <li>
-        <button onClick={this.toggle()} className="usa-accordion-button" aria-expanded={expanded} aria-controls="amendment-1">
+        <button onClick={this.toggle} className="usa-accordion-button" aria-expanded={expanded} aria-controls="amendment-1">
```
Comment (saneshark) [inline src/js/gi/components/AccordionItem.jsx]: Not sure what best practice is around this, but this looks almost like inline javascript. Is it intended? Wouldn't it be better to do some kind of live binding for a button having this class name?

File: src/js/gi/selectors/calculator.js [hunk 2]
```diff
@@ -9,15 +9,764 @@ const getEligibilityDetails = (state) => {
...
+    numberOfTerms,
+    tuitionNetPrice,
+    housingStipdendTerm1,
+    housingStipdendTerm2,
+    housingStipdendTerm3,
+    housingStipdendTotal,
+    tuitionAllowTerm1,
+    tuitionAllowTerm2,
+    tuitionAllowTerm3,
+    tuitionAllowTotal,
```
Comment (saneshark) [inline src/js/gi/selectors/calculator.js]: are these liable to change? I'm wondering if maybe these should be returned as a constant or something from the backend. Just a thought -- obviously for a future enhancement.

File: src/js/gi/components/AccordionItem.jsx (hunk 1 above)
Comment (typesend) [inline src/js/gi/components/AccordionItem.jsx]: Nope. Short answer: This is the React way.

File: src/js/gi/selectors/calculator.js (hunk 2 above)
Comment (typesend) [inline src/js/gi/selectors/calculator.js]: They are not constants. They are computed based on calculator form inputs and data from the backend (school attrs and constants). Basically what you are suggesting is akin to making the calculator a server-side activity requiring a round trip every time the user changes a form input.

**Comment by U-DON** [comment]
Closing this to open an intermediate PR.

**Pull request closed without merging**